api/bnd.bnd

```diff
@@ -1,5 +1,10 @@
 -exportcontents: \
     org.eclipse.microprofile.*
+
+Import-Package: \
```

Comment:
The version range should be handled in pom.xml not here.

Reply:
How would you propose handling it in pom.xml? Via the `org.apache.felix:maven-bundle-plugin` or something else?

Reply:
@arjantijms  yes, in maven-bundle-plugin. I did experiment a bit with pom.xml. It seems it is not as visible as specifying in the bnd.bnd. sorry, I had a false impression. What you have done is good. Actually, if cdi api was properly semantic versioned, we won't have this headache. This is why I campaign for MP to do semantic versioning properly. If you fix your ip-validation, your change can be merged in.